quickstart-services.yml

```diff
@@ -145,3 +149,19 @@ services:
       - type: bind
         source: ./.build/synapse
         target: /data
+
+  rabbitmq:
+    container_name: alkemio_dev_rabbitmq
+    image: docker.io/bitnami/rabbitmq:3.9
+    restart: always
+    networks:
+      - alkemio_dev_net
+    ports:
+      - 4369
+      - 5672
+      - 25672
```

Comment:
why also exposing 25672?

Reply:
It's not exposed outside of the network. It was the original set of ports required for the bitnami rabbitmq container.